spring-cloud-open-service-broker-core/src/main/java/org/springframework/cloud/servicebroker/model/ServiceInstanceMetadata.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.servicebroker.model;
```

Comment:
This should be in the `org.springframework.cloud.servicebroker.model.instance` package instead.

Reply:
Done. :)

Reply:
Thanks!